submissions/5mountains/popup/css/main.css

```diff
@@ -0,0 +1,291 @@
+    opacity: 1;
+}
+
+.menu__item-label {
+    width: 30px;
+    height: 30px;
+    display: block;
+    cursor: pointer;
+    background-image: url("../img/menu/popup-button.png");
+    background-size: cover;
+}
+
+.menu__item-label::before {
+    opacity: 0;
+    content: "";
+    z-index: 10;
+    position: absolute;
+    top: 60px;
+    right: 130px;
+    width: 10px;
+    height: 10px;
+    background-color: #fff;
+    border-top: 1px solid #ccc;
+    border-right: 1px solid #ccc;
+    transform: rotate(-45deg);
+}
+
+.menu__link-text,
+.menu__item-label,
+.menu__link-bell,
+.menu__link-user {
+    border: 1px solid transparent;
+}   
+
+.menu__link-text:focus,
+.menu__item-label:focus,
+.menu__link-bell:focus,
+.menu__link-user:focus {
+    border: 1px solid #00f;
+}   
+
+.menu__link-text {
+    padding: .5rem;
+}
+
+.menu__link-text:hover {
+    text-decoration: underline;
+    opacity: .75;
+}
+
+.menu__link-bell {
+    background-image: url("../img/menu/bell.png");
+    background-size: cover;
+    display: block;
+    width: 22px;
+    height: 22px;
+}
+
+.menu__link-bell:hover {
+    opacity: .75;
+}
+
+.menu__link-user {
+    background-image: url("../img/menu/user.png");
+    background-size: cover;
+    display: block;
+    width: 35px;
+    height: 35px;
+    border-radius: 50%;
+}
+
+.menu__link-user:hover {
+    box-shadow: 0 1px 0 rgba(0,0,0,.15);
+}
+/* 
+    Main menu style END
+*/
+
+
+
+/* 
+    Submenu of main menu style START
+*/
+.menu__submenu {
+    z-index: 5;
+    width: 320px;
+    max-height: 500px;
+    position: absolute;
+    display: none;
+    top: 65px;
+    right: 70px;
+    border: 1px solid #ccc;
+    z-index: 1;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    overflow-x: hidden;
+    padding: .5rem .5rem 1.6rem .5rem;
+    box-shadow: 3px 3px 10px rgba(0, 0, 0, .3);
+}
+
+.submenu__item {
+    height: 100px;
+    width: 100px;
+    display: flex;
+    align-items: flex-end;
+    margin-bottom: 1rem;
+}
+
+.submenu__link {
+    width: 100%;
+    height: 100%;
+    display: block;
+    text-align: center;
+    border: 1px solid transparent;
+}
+
+.submenu__link img {
+    width: 75%;
+}
+
+.submenu__link:hover {
+    border: 1px solid #ccc;
+}
+
+.submenu__link:focus {
+    border: 1px solid #00f; 
+}
+
+.menu__submenu [type="checkbox"] {
+    display: none;
+}
+
+/* scrollbar style start*/
+.menu__submenu::-webkit-scrollbar {
+    width: .15rem;
+}
+
+.menu__submenu::-webkit-scrollbar-track {
+    box-shadow: inset 0 0 5px rgba(0, 0, 0, 0.3);
+}
+
+.menu__submenu::-webkit-scrollbar-thumb {
+    background-color: rgb(182, 180, 180);
+    border: 1px solid rgb(58, 64, 70);
+}
+/* scrollbar style end*/
+
+.submenu__item-label {
+    position: absolute;
+    width: 100%;
+    bottom: 0;
+    left: 0;
+    background: #f1f1f1;
+    cursor: pointer;
+    padding: .4rem 0;
+    text-align: center;
+}
+/* 
+    Submenu of main menu style END
+*/
+
+
+/*
+    Extra menu of submenu style START
```

Comment:
Avoid committing commented code. Use another extra branch for this.

Reply:
Do you mean that I should use other files (for instance header.css, nav.css, menu.css, etc.) in order to separate code? Thanks for your review!

Reply:
No.
As the mentors told me that the final project should not contain any comments
If you want to leave comments for yourself, then for this it is better to use some other branch, like debug

Just don't create anything now, just clean up the comments.

Reply:
Wow, I didn't know that! I'll remove the comments!

Reply:
How about htm file? There are also some comments on the sections...

Reply:
I meant that you should always leave a blank line at the end of file.